App/Development.xcconfig

```diff
@@ -0,0 +1,14 @@
+//
+//  Development.xcconfig
```

Comment:
I assume this file is not needed

Reply:
Why? I thought you were showing us how to create it and we need it for a configuration struct.